### ISSUE TITLE
About module my-local-ip

### ISSUE THREAD

**Issue by thongpham281** [nanoscopic/stf-android-provider#1]
![Screenshot from 2021-01-29 15-34-26](https://user-images.githubusercontent.com/48922153/106251439-8d6f1700-6247-11eb-9815-f4f5d35234b5.png)
i don't have module my-local-ip when i pull this repo!
how can i config or pull that module?

**Comment by nanoscopic** [comment]
Yeah NodeJS modules change constantly and it has to be maintained endlessly. This is why I have abandoned the NodeJS code for the most part. When I have some spare time I'll get this running again but it isn't high on my priority list.

Please read https://github.com/DeviceFarmer/stf_ios_support/issues/94

This repo ( stf-android-provider ) was mainly/only created as a stepping stone to create stf-ios-provider. It is also useful for open source users of OpenSTF ( now DeviceFarmer ) to build and run an Android Provider without building the full codebase, but this repo has not been kept up to date to match upstream STF project.

The main project doesn't maintain this project ( and neither do I ). Really this project should be archived. *unsure*

At some point in the future I will integrate Android support into ControlFloor, and reccomend the community move away from DeviceFarmer/STF to ControlFloor entirely.

For the time being, until that happens, please use the upstream DeviceFarmer/STF project to create an Android provider.
